test/primitives.jl

```diff
@@ -179,13 +179,11 @@ module TestPrimitives
     @test dropnull(z) == [1, 2, 3, 5, 7]
 
     # dropnull(X::AbstractVector)
-    A = Any[Nullable(1), Nullable(2), Nullable(3), Nullable(), Nullable(5),
-            Nullable(), Nullable(7), Nullable()]
+    A = Any[1, 2, 3, Nullable(), 5, Nullable(), 7, Nullable()]
     @test dropnull(A) == [1, 2, 3, 5, 7]
 
     # dropnull(X::AbstractVector{<:Nullable})
-    B = [Nullable(1), Nullable(2), Nullable(3), Nullable(), Nullable(5),
-         Nullable(), Nullable(7), Nullable()]
+    B = Nullable[1, 2, 3, Nullable(), 5, Nullable(), 7, Nullable()]
```

Comment:
You don't even need `Nullable` here, as promotion will make this an `Array{Nullable{Int}}` automatically.